scripts/download.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import gzip
+import hashlib
+import os
+
+try:
+    import requests
+except ImportError:
+    raise RuntimeError("requests is required")
+
+
+def sha256(content):
+    m = hashlib.sha256()
+    m.update(content)
+    return m.hexdigest()
+
+
+def fetch_records(session, url, since=None):
+    if since:
+        url = url + "?_since=%s" % since
```

Comment:
You should add a ``_sort=-last_modified`` parameter here according to the way you store the timestamp.